tools/src/test/java/it/eng/tools/service/ApplicationPropertiesServiceTest.java

```diff
@@ -0,0 +1,78 @@
+package it.eng.tools.service;
+
+import it.eng.tools.exception.ApplicationPropertyErrorException;
+import it.eng.tools.model.ApplicationProperty;
+import it.eng.tools.repository.ApplicationPropertiesRepository;
+import it.eng.tools.util.MockObjectUtil;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.data.domain.Sort;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+public class ApplicationPropertiesServiceTest {
+
+	@Mock
+	private ApplicationPropertiesRepository repository;
+
+	@InjectMocks
+	private ApplicationPropertiesService service;
+
+	private ApplicationProperty property = MockObjectUtil.PROPERTY;
+
+	@Test
+
+	@DisplayName("Get application properties successfully") void
+	getApplicationProperties_success() {
+		when(repository.findAll(Sort.by("id"))).thenReturn(Collections.singletonList(property));
+		List<ApplicationProperty> propertiesList = service.getProperties(null);
+		assertNotNull(propertiesList);
+		//assertNotEquals(propertiesList.size(), 0);
```

Comment:
Why is this line commented?
Why not verify that size is 1, if MockObjectUtil.PROPERTY is indeed single test property